src/functions/functions-api.ts

```diff
@@ -58,6 +60,40 @@ export type TaskOptions = DeliverySchedule & TaskOptionsExperimental & {
    * The default is 10 minutes. The deadline must be in the range of 15 seconds and 30 minutes.
    */
   dispatchDeadlineSeconds?: number;
+
+  /**
+   * The id to use for the enqueued event.
+   * If not provided, one will be automatically generated.
+   * If provided explicitly specifying a task ID enables task de-duplication. If a task's ID is
+   * identical to that of an existing task or a task that was deleted or executed recently then
+   * the call will throw a TaskAlreadyExists error. Another task with the same id can't be
+   * created for ~1hour after the original task was deleted or executed.
+   * 
+   * Because there is an extra lookup cost to identify duplicate task ids, setting id
```

Comment:
Yup, please do a scrub throughout for "ID," which should be capped -- unless you're referring to a lowercase literal, in which case it should be backticked.  Thanks!